lms/templates/instructor/instructor_dashboard_2/certificates.html

```diff
@@ -92,6 +92,25 @@
         %endif
     % endif
 
+    <hr>
+    <p class="start-certificate-regeneration">
+        <h2>${_("Regenerate Certificates")}</h2>
+        <form id="certificate-regenerating-form" method="post" action="${section_data['urls']['start_certificate_regeneration']}">
+            <p id='status-multi-select-tip'>${_('Select one or more students below using your mouse and ctrl or command key.')}</p>
```

Comment:
nit: this label should say `Select one or more certificate statuses below using your mouse and ctrl or command key.`